configuration/export.go

```diff
@@ -11,11 +11,28 @@ package configuration
 type Exporter struct {
```

Comment:
file have name export, why sthis object have different name?

Reply:
I haven't any idea, why it was before, but I will rename the file.

Reply:
File has been renamed.

Reply:
thx